test/atom/pane-item.test.js

```diff
@@ -89,7 +89,7 @@ describe('PaneItem', function() {
   describe('when opened with a matching URI', function() {
     it('calls its render prop', async function() {
       let called = false;
-      shallow(
+      mount(
```

Comment:
The test started failing, so I'm guessing there was a change to Enzyme's shallow renderer such that it no longer supports render props the same way. I didn't look into it _that_ hard because just changing it to `mount` worked.